src/lib/Scenes/Collection/Screens/CollectionHeader.tsx

```diff
@@ -54,7 +54,7 @@ export const CollectionHeaderContainer = createFragmentContainer(CollectionHeade
         edges {
           node {
             image {
-              url
+              url(version: "larger")
```

Comment:
How do the dimensions of the `larger` image compare to what we need to display in the app?

Asking in case we want to run this through a resizer as well. 

(But I recognize this might be complicated by variation in phone/tablet screen sizes and Retina, etc)

Reply:
Answering my own question by peeking into a Gemini console:

```ruby
gemini:production> pp Version.find_by(key: "larger")
#<Version:0x000055bfd71a1c28
 id: 20,
 height: 1024,
 width: 1024,
 quality: 81,
 resize_to: "limit",
 key: "larger",
 description: "Larger Images",
 created_at: Mon, 06 Oct 2014 22:11:05 UTC +00:00,
 updated_at: Wed, 12 Jun 2019 18:02:59 UTC +00:00,
 convert_to: "jpg",
 resample: true,
 min_resampled_size: 1025,
 resampled_resolutions: [72, 150],
 strip_metadata: true,
 deep_zoom: false,
 skip_resize: false,
 extract_visual_genome: false,
 watermark: false>
```

Reply:
This^ sounds reasonable to me. I'll go ahead and merge — we can follow up if anyone thinks that's necessary.  Thanks for the fix 🙌 !

Reply:
We settled on `larger` through experimentation, so I appreciate you digging deeper on this.